pkg/arvo/app/btc-provider.hoon

```diff
@@ -327,21 +334,24 @@
 ~%  %btc-provider-helper  ..card  ~
 |_  =bowl:gall
 ++  send-status
-  |=  =status
+  |=  [=status ship=(unit ship)]
   ^-  card
   %-  ?:  ?=(%new-block -.status)
         ~&(>> "%new-block: {<block.status>}" same)
       same
-  [%give %fact ~[/clients] %btc-provider-status !>(status)]
+  =-  [%give %fact ~[-] %btc-provider-status !>(status)]
+  ?~  ship  /clients
```

Comment:
Do wires work that way? I've never seen this before.

Reply:
Yep! You can look at `/lib/push-hook` for more examples.